src/poll/CreatePoll.js

```diff
@@ -7,18 +8,22 @@ const STATUS_CREATING = 'Creating...'
 const STATUS_SUCCESS = 'Your poll was created successfully'
 const STATUS_FAIL = 'We failed to create your poll :(' 
 
-class Poll extends Component{
+class CreatePoll extends Component{
 
   state = this.defaultState()
 
   defaultState(override={}) {
```

Comment:
It might be clearer to rename this method to `createInitialState` and the `override` param to `state`.